<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NodeManager</class>
 <widget class="QMainWindow" name="NodeManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>940</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>791</width>
      <height>501</height>
     </rect>
    </property>
    <property name="title">
     <string>游戏UI结构</string>
    </property>
    <widget class="QTreeWidget" name="treeWidget_node_tree_">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>771</width>
       <height>471</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <attribute name="headerVisible">
      <bool>true</bool>
     </attribute>
     <attribute name="headerDefaultSectionSize">
      <number>100</number>
     </attribute>
     <column>
      <property name="text">
       <string>节点</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>类型</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>名称</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>UUID</string>
      </property>
     </column>
     <item>
      <property name="text">
       <string>director</string>
      </property>
      <property name="text">
       <string>Director</string>
      </property>
      <property name="text">
       <string>director</string>
      </property>
      <property name="text">
       <string>111</string>
      </property>
      <item>
       <property name="text">
        <string>world_scene</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <item>
        <property name="text">
         <string>map_jpg</string>
        </property>
       </item>
      </item>
      <item>
       <property name="text">
        <string>battle_scene</string>
       </property>
       <item>
        <property name="text">
         <string>units</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>effect</string>
        </property>
       </item>
      </item>
     </item>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>配置文件</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_config_file">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>561</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_load_config_file">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>20</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_create_child_node">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>120</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>创建子节点</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_undo">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>480</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>撤回</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_node_type_list">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>60</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_create_config_file">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>20</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>创建配置文件</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_save">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>510</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>保存</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_node_name_input">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>90</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_modify_node_attr">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>150</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>更改当前节点属性</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_delete_node">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>180</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>删除节点</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>940</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="actionAbout_me"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout_me">
   <property name="text">
    <string>About me...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
